src/compiler/types.ts

```diff
@@ -8218,6 +8218,11 @@ export type EmitHelperUniqueNameCallback = (name: string) => string;
  * @internal
  */
 export const enum LanguageFeatureMinimumTarget {
+    // ES3 Features
+    RegularExpressionFlagsGlobal = ScriptTarget.ES3,
+    RegularExpressionFlagsIgnoreCase = ScriptTarget.ES3,
+    RegularExpressionFlagsMultiline = ScriptTarget.ES3,

```

Comment:
Probably these should have been unconditional; we should not be referencing this script target anywhere anymore but the original regex PR predated us removing ES3 so that's just how it is.

Reply:
lets make these unconditional and remove the target here given ES3 is deprecated.